flight/mcu/lib/teensyShared/servoControl.h

```diff
@@ -19,8 +19,18 @@ struct servo_PNTR {
     datalogger_THD *lowgDataloggerTHDVarsPointer;
 };
 
-void servoTickFunction(pointers *, PWMServo *, PWMServo *);
-
-void round_off_angle(int &value);
+class ServoControl {
+    public:
+        ServoControl(struct pointers* pointer_struct);
+
+        void servoTickFunction(pointers *, PWMServo *, PWMServo *);
+
+    private:
```

Comment:
In the `private:` section, lets add pointers to PWMServo objects like
```
PWMServo* servo_cw;
PWMServo* servo_ccw;
```
and initialize these during construction

This way, we won't need to pass those pointers into the `servoTickFunction()` every time.